src/GafferCycles/IECoreCyclesPreview/Renderer.cpp

```diff
@@ -991,6 +992,7 @@ class CyclesAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 
 			// Light shader
 
+			m_muteLight = optionalAttribute<bool>( g_muteLightAttributeName, attributes );
```

Comment:
We don't need to use `optional` here, since we never do anything other than fall back to `false` anyway. We can simplify a little and just do `m_muteLight = attributeValue<bool>( g_muteLightAttributeName, attributes, false )` into a `bool m_muteLight`.

Reply:
Fixed in 2a515fc072bed38621490420662667d98e32d6ff